.github/workflows/backend-cd.yml

```diff
@@ -0,0 +1,28 @@
+name: backend-cd
+
+on:
+  push:
+    branches: [main]
+    paths: [backend/**]
+  pull_request:
```

Comment:
I'm pretty sure we don't want to deploy a new version for pull requests, only after they are merged (which is what the `push:` trigger does)

Reply:
You are free to merge now